gui/wxpython/iscatt/plots.py

```diff
@@ -838,6 +837,13 @@ def _deleteVertex(self, event):
 
         self.Redraw()
 
+    def _dist_point_to_segment(p, s0, s1):
+        p = np.asarray(p, float)
+        s0 = np.asarray(s0, float)
+        s1 = np.asarray(s1, float)
+
+        return np.cross(s1 - s0, p - s0) / np.linalg.norm(s1 - s0)
```

Comment:
It can return negative numbers, so you need np.abs(...), that's mentioned also in the SO answer.